jetty-quickstart/src/main/java/module-info.java

```diff
@@ -18,4 +18,5 @@
     requires jetty.servlet.api;
     requires transitive org.eclipse.jetty.annotations;
     requires org.slf4j;
+    requires org.eclipse.jetty.servlet;
```

Comment:
@sbordet again, maybe these are transitive via org.eclipse.jetty.annotations?

Reply:
Not needed, remove it.